kubernetes/omsagent.yaml

```diff
@@ -634,7 +634,7 @@ spec:
               value: "VALUE_USER_ASSIGNED_IDENTITY_CLIENT_ID_VALUE"
             # Add the below environment variable to true only in sidecar enabled regions, else set it to false
             - name: SIDECAR_SCRAPING_ENABLED
-              value: "false"
+              value: "true"
```

Comment:
Rashmi, does this mean we can enable side-car scraping in the chart as well (for non-aks) ?

Reply:
Yes Vishwa. I am updating this since AKS mostly has this change rolled out in all regions barring sovereign clouds and wanted our testing to include these changes. i will create a task to update our chart as well.